tests/e2e_plugable_policies_test.go

```diff
@@ -23,6 +24,9 @@ var _ = Describe("ApplicationAwareQuota plugable policies", func() {
 	Context("label-sidecar evaluator", func() {
 		ctx := context.Background()
 		BeforeEach(func() {
+			if !strings.Contains(f.KubeConfig, "application-aware-quota/_ci-configs") {
+				Skip("label-sidecar exist only in the ci environment")
+			}
```

Comment:
Is there a way to dynamically check that? This code is heavily dependent on our specific CI environment

Reply:
That's a good point, thanks for noting this.

Fixed :)